src/utils.jl

```diff
@@ -1218,11 +1218,15 @@ end
 const _attrsymbolcache = Dict{Symbol, Dict{Symbol, Symbol}}( 
     :x => Dict( k => Symbol("x", String(k)) for k in keys(_axis_defaults)),
     :y => Dict( k => Symbol("y", String(k)) for k in keys(_axis_defaults)),
-    :z => Dict( k => Symbol("z", String(k)) for k in keys(_axis_defaults))
+    :z => Dict( k => Symbol("z", String(k)) for k in keys(_axis_defaults)),
+    # for PyPlot
+    :label => Dict( k => Symbol("label", String(k)) for k in (:top, :bottom, :left, :right)),
```

Comment:
IMHO, the PR was more readable before , this is more difficult to maintain.

Reply:
Note that using `_axis_defaults` in `utils.jl` creates a circular dependency between `utils.jl` and `args.jl`.

Reply:
and if you move this to `args.jl`?

Reply:
This is horrible, but
```julia
julia> GC.enable(false); @time main(); GC.enable(true); GC.gc()
6.855578 seconds (10.66 M allocations: 627.726 MiB)
julia> GC.enable(false); @time main(); GC.enable(true); GC.gc()
  1.124249 seconds (2.49 M allocations: 150.345 MiB)
```j
First: slower, more allocs. Repeated: similar.

Reply:
Thats somewhat surprising to me, but then it would be good to revert to the dynamic caching

Reply:
Done